### ISSUE TITLE
[Bug Report] Limit expanded content width to width of data table

### ISSUE THREAD

**Issue by lsapan** [vuetifyjs/vuetify#3628]
### Versions and Environment
**Vuetify:** 1.0.9
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.162
**OS:** Mac OS 10.13.3

### Steps to reproduce
Go to the Codepen and observe the test-content above the table. It is a simple 3 column layout, with a list in the middle. The list has one item, and that item has a very long subtitle, which is properly truncated.

Expand the first table row, a "short" version of that content is used, and you can see the subtitle is properly truncated and everything displays properly.

Expand the second (or any other) table row, the same content that is used above the table is inserted here, meaning the longer subtitle is used. Instead of properly truncating it, it makes the expanded content much wider. Interestingly, the subtitle IS still truncated, but it's much wider than it should be.

### Expected Behavior
The second row should be the same width as the first row when expanded.

### Actual Behavior
The second row's expansion forces the table to be much wider, and stretches everything out.

### Reproduction Link
[https://codepen.io/anon/pen/bvgydM](https://codepen.io/anon/pen/bvgydM)

### Other comments
Removing `fluid` from the v-container fixes the issue **(update: not always)**, but we need the expanded content to fill the width of the table.

**Comment by praveenpuglia** [comment]
To me it seems like the problem is coming from the `white-space: nowrap` applied to the `.list__tile__title` & `list__tile__sub-title`.  Setting `white-space: normal` fixes your issue.

Going from those elements to the top in the DOM tree, none of the elements have explicit width and the td has to accommodate that content so it expands and allows things to expand.

I may still be wrong though.

**Comment by lsapan** [comment]
@praveenpuglia good catch, setting `white-space: normal` fixes the width issue, but then the text takes up more than one line. Something like this will work for now:

```
.list__tile__sub-title, .list__tile__title {
    white-space: normal;
    height: 24px;
}
```

It doesn't properly truncate with an ellipsis, but it's certainly better than the alternative! That said, I'm not sure why the browser is doing that to begin with. I found [this SO question](https://stackoverflow.com/questions/34934586/white-space-nowrap-and-flexbox-did-not-work-in-chrome) which seems to be related, but I tried applying the `min-width: 0` fix and it didn't seem to make a difference.
